examples/ITK_Example_RegistrationOfPointsInMemory.ipynb

```diff
@@ -0,0 +1,190 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f9aa2e00-18c3-4753-be93-9d7c2ef5d0b9",
+   "metadata": {},
+   "source": [
+    "## Registration of Points in memory"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2cd37425",
+   "metadata": {},
+   "source": [
+    "Demonstrates how to use the points of a pair of itk.PointSet objects as input to a registration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "639d3356",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import itk\n",
+    "\n",
+    "# Generate the corner points of a square\n",
+    "def make_square(origin: tuple):\n",
+    "\n",
+    "    origin_x = origin[0]\n",
+    "    origin_y = origin[1]\n",
+    "    point_set = itk.PointSet[itk.UC, 2].New()\n",
+    "\n",
+    "    for x in (0, 1):\n",
+    "        for y in (0, 1):\n",
+    "            point_set.SetPoint(point_set.GetNumberOfPoints(), (origin_x + x, origin_y + y))\n",
```

Comment:
`point_set.SetPoint(point_set.GetNumberOfPoints(), ...)` is a bit cumbersome, but PointSet has no AppendPoint method. We could use `push_back(...)` here instead, when we access the internal `std::vector` that stores the points. Something like:

    points = point_set.GetPoints().CastToSTLContainer() 
    points.push_back(...)